src/applications/claims-status/utils/helpers.js

```diff
@@ -302,7 +302,7 @@ export function makeAuthRequest(
 }
 
 export function getCompletedDate(claim) {
-  if (claim.attributes && claim.attributes.eventsTimeline) {
+  if (claim?.attributes && claim.attributes.eventsTimeline) {
```

Comment:
This might be able to be reduced to:

```js
   if (claim?.attributes.eventsTimeline) { 
```

Reply:
I kept getting `null` errors on this one, even with the optional chaining... but it must have been a fluke because it works now.